scripts/clean_project.sh

```diff
@@ -7,5 +7,6 @@ echo "Cleaning XpringKit Project"
 rm -rf *.xcodeproj
 rm -rf XpringKit/generated
 rm -f XpringKit/Resources/bundled.js
```

Comment:
You can remove line 9. 

You could also just genericize this and make it  `rm-rf XpringKit/Resources` and not have to think about this again.

Reply:
Smart, made that change :)